api/api.proto

```diff
@@ -2163,24 +2171,32 @@ message SdkCloudBackupStatus {
   google.protobuf.Timestamp completed_time = 6;
   // NodeID is the ID of the node where this Op is active
   string node_id = 7;
+  // SourceVolumeID is the the volume that is either being backed up to cloud
+  // or target volume to which cloudsnap is being restored
```

Comment:
What is cloudsnap? I think that is a Px word, but what is the similar technology in OpenStorage?

Reply:
Okay.